features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Documents/Edit.jsx

```diff
@@ -145,7 +145,7 @@ function Edit(props) {
                             id='Apis.Details.Documents.Edit.documents.text.editor.edit.content'
                             defaultMessage='Edit '
                         />
-                        "{props.docName}"
+                        &nbsp;"{props.docName}"
```

Comment:
Let's use string [Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) to construct a string with spaces etc 

I:e 

```js
`${props.docName} ${props.docName}`
```

So that React will do the encoding automatically

Reply:
fixed with https://github.com/wso2/carbon-apimgt/pull/10240/commits/5dbf71fadb466cb6b28031b3cdf322014ea3afcd